src/portal/Portal.vue

```diff
@@ -21,29 +15,7 @@ import { Component, Vue, Watch } from "vue-property-decorator";
   components: { FundsCard },
```

Comment:
Can we remove the unused component `FundsCard`?
and the unused imports

Reply:
done.

Reply:
can we remove this line? and its import

Reply:
done.